lib/querystring.js

```diff
@@ -354,7 +354,7 @@ function parse(qs, sep, eq, options) {
               encodeCheck = 1;
               continue;
             } else if (encodeCheck > 0) {
-              if (isHexTable[code] === 1) {
+              if (isHexTable[code]) {
```

Comment:
Nit: I'd prefer to leave these as is to avoid number->boolean casting.

Reply:
Make sense, I agree. I fix it and force-push.